pkgs/by-name/po/podman-bootc/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  libisoburn,
+  libvirt,
+  pkg-config,
+}:
+
+buildGoModule rec {
+  pname = "podman-bootc";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "containers";
+    repo = "podman-bootc";
+    rev = "v${version}";
+    hash = "sha256-Hxg2QSedPAWYZpuesUEFol9bpTppjB0/MpCcB+txMDc=";
+  };
+
+  vendorHash = "sha256-8QP4NziLwEo0M4NW5UgSEMAVgBDxmnE+PLbpyclK9RQ=";
+
+  tags = [
+    "exclude_graphdriver_btrfs"
+    "btrfs_noversion"
+    "exclude_graphdriver_devicemapper"
+    "containers_image_openpgp"
+    "remote"
+  ];
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [
+    libvirt
+    libisoburn
+  ];
+
+  # All tests depend on booting virtual machines, which is infeasible here.
+  doCheck = false;
+
```

We should also install the shell completion. The emulator stuff will make it work when cross compiling (and when possible) as well

```suggestion

  postInstall =
    let
      podman-bootc = "${stdenv.hostPlatform.emulator buildPackages} $out/bin/podman-bootc";
    in
    lib.optionalString (stdenv.hostPlatform.emulatorAvailable buildPackages) ''
      installShellCompletion --cmd podman-bootc \
        --bash <(${podman-bootc} completion bash) \
        --fish <(${podman-bootc} completion fish) \
        --zsh <(${podman-bootc} completion zsh)
    ''; 
```

---

Nice, didn't see any completions in upstream but they use https://pkg.go.dev/github.com/spf13/cobra.